src/main/kotlin/io/perfometer/dsl/HttpDsl.kt

```diff
@@ -18,8 +18,11 @@ class RequestBuilder(
     private val headers: MutableList<() -> HttpHeader> = mutableListOf()
     private val params: MutableList<() -> HttpParam> = mutableListOf()
 
-    val response = HttpResponse()
     var body: () -> ByteArray = { ByteArray(0) }
+        private set
+
+    var process: (response: HttpResponse) -> Unit = {}
```

Comment:
Consider renaming `process` to `processor` or `responseProcessor`, or simillar. I would suggest establishing a convention for function types in line with what's in JDK, e.g. Supplier/Producer/Consumer.

Reply:
Good idea, I even think naming it consumer here would be better. Still generic but more clear to me than process(or).